java/src/com/ibm/streamsx/topology/context/remote/RemoteContext.java

```diff
@@ -16,7 +16,8 @@
     public enum Type {
         TOOLKIT,        
         BUILD_ARCHIVE,
-        REMOTE_BUILD_AND_SUBMIT
+	REMOTE_BUILD_AND_SUBMIT,
```

Comment:
The StreamsContextFactory doesn't have access to the config object when generating Contexts:
```Java
public static StreamsContext<?> getStreamsContext(StreamsContext.Type type)
```
If we modify it such that it accepts the config object:
```Java
public static StreamsContext<?> getStreamsContext(StreamsContext.Type type, Map<String, Object> config)
```
It will break the backwards compatibility of getStreamsContext.